src/lib.rs

```diff
@@ -290,44 +281,41 @@ impl<R> ResUnit<R>
 where
     R: gimli::Reader,
 {
-    fn parse_lines(&self, sections: &DebugSections<R>) -> Result<&Lines<R>, Error> {
+    fn parse_lines(&self) -> Result<Option<&Lines<R>>, Error> {
+        let ilnp = match self.dw_unit.line_program {
+            Some(ref ilnp) => ilnp,
+            None => return Ok(None),
```

Comment:
This [used to be](https://github.com/gimli-rs/addr2line/pull/115/files#diff-b4aea3e418ccdb71239b96952d9cddb6L218) a straight `unreachable!()`, so I believe it should be extremely uncommon. From this perspective returning an error here *may* be warranted (if the line program for my unit is missing entirely, I probably want to know that instead of just being told "no idea where this address maps"). Not sure though.

Reply:
I'm not sure either. Returning `None` lets us get any function information if available. But yes, it should be extremely uncommon.